### ISSUE TITLE
Cooperative hunting

### ISSUE THREAD

**Issue by skrohmagnon** [jeremylarner/keyteki#362]
It's doesn't work properly. You can slip damages between creatures but now you can't do more damage than its power to a creature. All damages suppose to be instant so you can't kill shadow self the creature protected by it. But now you can.

**Comment by MooDySmiles** [comment]
I don't think that the damage is supposed to be instant :thinking: 
Text says: "Deal 1 for each friendly creature in play. You may divide this damage among any number of creatures.", so you deal 1 damage at time for each friendly creature, it's impossible to deal more damage than the power of a creature.
I'm not playin' since a couple of months, but I remember clearly this is the behaviour this card has and how it's supposed to be...

**Comment by skrohmagnon** [comment]
rulebook p10 : "If multiple creatures are damaged by a single effect, that damage is dealt simultaneously"
It's clear. You can do more than its power on a creature. The second sentence explain it : you "may" split the damages between creatures.

**Comment by MooDySmiles** [comment]
Uhm, you may be right, but Cooperative Hunting is not your typical AoE card, it can be interpreted in different ways :smiley:

**Comment by skrohmagnon** [comment]
there is no aoe cards. There is only cards which deals damages to more than one creature. And the rules are clear with this case. If the text says "deal 1 damage, repeat this effect for each creature you control" i would be ok ith you but it is not the case. So it's one effect. You can overkill a creature and damages are dealt simultaneously (p10)

**Comment by MooDySmiles** [comment]
Yes, now I understand. You are completely right, the behaviour should be corrected!

**Comment by JEBailey** [comment]
I understand the rule in question. I don't understand what problem it's causing. In particular with the shadow self card.

**Comment by skrohmagnon** [comment]
At this moment, the damages are dealt one after one so you so if you have a 1 power creature protected by a shadow self with 8 damages, you can kill the sadow self with one damage then kill the 1-power creature. But damages are dealt simultaneously so you should kill only the shadow self.

**Comment by JEBailey** [comment]
@skrohmagnon I appreciate the clarification. So the ask to resolve this, is that creatures that are protected by shadowself can't be legitimate targets of cooperative hunting. This is is technically incorrect as you can target one of these creatures in the game and the damage just goes to the shadow self, but it's the only way I could see that working in an online scenario.

**Comment by skrohmagnon** [comment]
there is still the problem of overkill which is not possible now. Can you check the destruction of creatures at the end of the effect ? 
1 deal damages as now
2 check if a creature has more demages than its power 
it could solve the problem with shadow self too.
